<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelImportDockWidget</class>
 <widget class="QDockWidget" name="ModelImportDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Importer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Model Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pathDisplay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="searchBar">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxCount">
         <number>2147483642</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clearButton">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Directory</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
